src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -108,15 +99,9 @@ public function getQueryBuilderPartsForCachingHash($queryBuilder)
         return false;
     }
 
-    public function __construct(ManagerRegistry $registry, PropertyAccessorInterface $propertyAccessor = null, ChoiceListFactoryInterface $choiceListFactory = null)
+    public function __construct(ManagerRegistry $registry)
```

Comment:
I don't think we should remove this since it would be a BC break for anyone using the Form component and the DoctrineBridge without the full stack.